elasticsearch/templates/statefulset.yaml

```
@@ -143,6 +143,9 @@ spec:
       {{- if .Values.imagePullSecrets }}
       imagePullSecrets:
 {{ toYaml .Values.imagePullSecrets | indent 8 }}
+      {{- end }}
+      {{- if semverCompare ">1.13" .Capabilities.KubeVersion.GitVersion }}
+      enableServiceLinks: false
```

Note that we still want it to be configurable and enabled by default in `values.yaml`in addition of the K8S version check:

```suggestion
      enableServiceLinks: {{ .Values.enableServiceLinks }}
```

---

Ok, I made the value configurable and updated both values.yaml and README.md.